lms/assets.ini

```diff
@@ -1,14 +1,8 @@
 [bundles]
 
-file_picker_js =
-  scripts/file_picker.bundle.js
-
 frontend_apps_js =
   scripts/frontend_apps.bundle.js
 
-content_item_selection_js =
-  scripts/content_item_selection.bundle.js
-
```

Comment:
There are references to these bundles in `gulpfile.js` which should also be removed.

Reply:
Done, thanks